packages/astro/src/core/config.ts

```diff
@@ -346,6 +346,15 @@ export async function validateConfig(
 			adapter: undefined,
 		},
 	};
+	if (
+		// TODO: expose @astrojs/mdx package
```

Comment:
Is this a TODO now (in this PR) or in the future?

Reply:
@nate is planning to release it today, from what I can gather

Reply:
Great question @matthewp! Coming in a follow-up PR today.